docs/victory-area/docs.md

```diff
@@ -139,10 +139,19 @@ The `domainPadding` prop specifies a number of pixels of padding to add the begi
 
 ### label
 
+**DEPRECATED** use `labels` instead
+
 The `label` prop defines the label that will appear at the edge of the area. This prop should be given a string or as a function of data. If individual labels are required for each data point, they should be created by composing `VictoryArea` with `VictoryScatter`.
```

Comment:
does the _they should be created by composing `VictoryArea` with `VictoryScatter`..._ still apply to `labels`?